features/default_backend.feature

```diff
@@ -0,0 +1,49 @@
+        @sig-network @conformance @release-1.19
+Feature: Default backend
+  An Ingress with no rules sends all traffic to a single default backend.
+  The default backend is typically a configuration option of the
+  Ingress controller and is not specified in your Ingress resources.
+
+  If none of the hosts or paths match the HTTP request in the
+  Ingress objects, the traffic is routed to your default backend.
+
+    Rules:
+    - Response status code is 404.
+    - Response body contains arbitrary text.
+
+        Scenario: Ingress with host and no backend serviceName
+            Given a new random namespace
+              And reading Ingress from manifest "scenarios/001/ing.yaml"
```

Comment:
I'd remove these manifest references since they are pointing to nowhere in this repo.

Reply:
This was just an example to show how functions with parameters are generated. In fact, all this file is an example to show how the code generator works.

Reply:
Then maybe add a disclaimer to the Scenario name? 
```
Scenario: Sample scenario for code-generation -- Ingress with host and no backend serviceName
```

Reply:
done